tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -22,8 +23,18 @@ class AuthorizationCheck(APIView):
     def get(self, request, *args, **kwargs):
         """Handle get request and verify user is authorized."""
         user = request.user
+        logger.info("AUTH_CHECK")
+        logger.info(str(user))
         serializer = UserProfileSerializer(user)
+
         if user.is_authenticated:
+            if user.inactive_account:
+                logout(request)
+                response = Response({"authenticated": False, "inactive": True})
+                response.delete_cookie("id_token")
+                logger.info("Auth check FAIL for INACTIVE user on %s", timezone.now())
+                return response
```

Comment:
If a user logs in successfully but their account is deemed inactive by our system, `logout` the initial request, and return a new `Response` indicating the user is `inactive`.